internal/cmd/stack/local_preview.go

```diff
@@ -119,12 +119,27 @@ func localPreview() cli.ActionFunc {
 			requestOpts = append(requestOpts, graphql.WithHeader(internal.UserProvidedRunMetadataHeader, cliCtx.String(flagRunMetadata.Name)))
 		}
 
-		if err := authenticated.Client.Mutate(ctx, &triggerMutation, triggerVariables, requestOpts...); err != nil {
+		if err = authenticated.Client.Mutate(ctx, &triggerMutation, triggerVariables, requestOpts...); err != nil {
 			return err
 		}
 
 		fmt.Println("You have successfully created a local preview run!")
 
+		if cliCtx.Bool(flagPrioritizeRun.Name) {
+			var prioritizeMutation setRunPriorityMutation
+			variables := map[string]interface{}{
+				"stackId":    graphql.ID(stack.ID),
+				"runId":      graphql.ID(triggerMutation.RunProposeLocalWorkspace.ID),
+				"prioritize": graphql.Boolean(true),
+			}
+
+			if err = authenticated.Client.Mutate(ctx, &prioritizeMutation, variables); err != nil {
+				return err
```

Comment:
We should probably print the error and advice the caller to do this himself instead of failing outright. The run itself got created just fine, no reason to exit.

Reply:
f6fcd7349259762110b8a9782db8683fda2e3b8b